src/Auth0Client.ts

```diff
@@ -111,7 +111,7 @@ export default class Auth0Client {
       stateIn,
       nonceIn,
       code_challenge,
-      window.location.origin
+      this.options.redirect_uri || window.location.origin
```

Comment:
default `redirect_uri` was `window.location.origin`?

Reply:
just for web_message requests (that don't actually redirect anywhere)